### PR TITLE
Load Weather API information

### PR DESCRIPTION
**Pull request by ethanjurman**
Loads the API requests through the server. Then **through our server** we can make a request like so:
`localhost:3000/weather/43.2115/-77.6008`
This hides the API key from external users and we can make specific requests; we use the lat lng provided by the `Weather` Component and store the response into `Weather` state.

for running various API's make a directory `keys/` and place in it:
`weather-key` - key for http://api.openweathermap.org/

image

### CODE REVIEW

File: src/client/js/components/weather.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+export default class WeatherComponent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      temp: null
+    }
+  }
+
+  // kelvin to fahrenheit
+  k2f(kelvin,sigfigs){
```
Comment (CShapleigh) [inline src/client/js/components/weather.js]: Bad function name please change

Comment (CShapleigh) [inline src/client/js/components/weather.js]: Literally unmergeable

File: src/client/js/components/weather.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+export default class WeatherComponent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      temp: null
+    }
+  }
+
+  // kelvin to fahrenheit
```
Comment (CShapleigh) [inline src/client/js/components/weather.js]: this comment is pretty useless

Comment (mlw6607) [inline src/client/js/components/weather.js]: ^ this one too

**Comment by CShapleigh** [comment]
:100: / :100:

**Pull request merged**